ground_truth_slam/plugin.cpp

```diff
@@ -28,7 +28,7 @@ class ground_truth_slam : public plugin {
 	}
 
 	void feed_ground_truth(switchboard::ptr<const imu_cam_type> datum) {
-		ullong rounded_time = datum->dataset_time;
+		ullong rounded_time = datum->time.time_since_epoch().count();
```

Comment:
`_m_sensor_data` uses dataset time as the key, not time since start, so we need to add dataset time back in. There are plans to retire `ground_truth_slam` in the future and move that responsibility to a combined pose_prediction/pose_lookup plugin, but until then we need to keep dataset time around.

Reply:
Actually, never mind. Just record `dataset_first_time` like `pose_lookup` does and add it to the passed in time to get the correct key.